src/ui/public/toggle_panel/toggle_panel.js

```diff
@@ -12,7 +12,8 @@ app.directive('togglePanel', function () {
     template: template,
     scope: {
       togglePanelId: '@',
-      text: '@',
+      toggleButtonLabel: '=',
```

Comment:
Hmm... I was thinking in terms of identifying parts of the UI. When I look at the UI I've built and I'm identifying what I've created, I see a button and a panel. If I want to change the text of the button, I'll think "I want to change that button's text". So something referring to that button in the interface would be intuitive to me. How about `buttonText`?